### DISCUSSION TITLE
11.1.0

### DISCUSSION THREAD

**Discussion by SubJunk** [UniversalMediaServer/UniversalMediaServer#3058] [General]
This release extends the playlist API, improves performance and fixes bugs.

### General
- Improved caching of web interface files
- Added getServerPlaylists to playlist API
- Fixed broken thumbnail-url for playlist folder resolved by DbIdResourceLocator
- Added support for more language characters on web interface
- Updated and removed old links on About tab
- Fixed database fields truncating to less than the limit
- Fixed rare and unpredictable bugs

### Translation updates via Crowdin
- Catalan (99%) (thanks, Toni Grau i Quellos!)
- French (100%) (thanks, Archaos!)
- Polish (100%) (thanks, Karol Szastok!)

### Dependencies
- Update dependency com.puppycrawl.tools:checkstyle to v10.3
- Update dependency org.apache.maven.plugins:maven-pmd-plugin to v3.17.0
- Update surefire-version to v3.0.0-M7
- Bump com.sun.xml.bind-version from 3.0.2 to 4.0.0
- Bump junrar from 7.5.1 to 7.5.2

<hr /><em>This discussion was created from the release <a href='https://github.com/UniversalMediaServer/UniversalMediaServer/releases/tag/11.1.0'>11.1.0</a>.</em>

**Comment by MTOakey** [comment]
Here was one other thing I have not been able to figure out ...
Web page tab "title" on hover ... VERY easy in plain HTML, but THIS little bugger is evasive... 
I can "Inspect" with Developer tools in Chrome, make the change I want in/on the TAB - looks Great - but does not "keep" or stay the way I want, obviously, since THAT doesn't actually write to the file, and since the pages are dynamically created also, as they don't even exist.
There is nowhere I see in any of the 3 web...css files, but it seems the place to make this change I desire is in the "browse.html.orig" file, right there, in Line 37. The TAG <title>...</ttitle>. Currently there resides a variable {{name}} there.
I replaced that variable with what I want as the name/title of my tabs... and nothing changed, it didn't work.
I only wish for the folders I am sharing, only 2 right now and probably ever, one named "Videos", the other "Music", and I just want them to have the same title/name **I already input in the "Main Panel" > General Settings tab > :Server name: field**. THAT should make this type of "change" easy, a check box option that asks: "Use this as Browser Tab Title also" or something equally as creative.
My "Video" shared folder is page "browse/52" and the "Music" folder is page "browse/53"... the links that reside on page "browse/0" that lead to those respective folders read: "Video" and "Music" which is fine, that does not have to change.
The page "browse/52' is really the only page I would like to change.

I love automation as much as the next guy... but THIS has been driving me nuts for days now. I hate dynamically created pages - especially when unnecessary.

Any help or direction is and will be GREATLY appreciated.
